### DISCUSSION TITLE
Questions about some new features in fastDreambooth colab notebook

### DISCUSSION THREAD

**Discussion by i-huzaifa-arshad** [TheLastBen/fast-stable-diffusion#1516] [General]
So I am back to SD after a few weeks and I see there are some new feature in the notebook, that I want to ask. 

1. In Model Download cell, there are two options for model select. Model_Version make sense to me but what does the Custom_Model_Version do? 
![image](https://user-images.githubusercontent.com/76521534/215760555-310cb25b-099e-44fb-be7b-df0e25bc6230.png)
 2. What does this Captions tool do? 
![image](https://user-images.githubusercontent.com/76521534/215760915-668c28c2-2e96-4568-ac3d-4d42b6b29c9e.png)

**Comment by Excalibro1** [comment]
The first `model_version` is used if you don't enter a `huggingface path/share link/gdrive path` and will download the model from the original repository's but if you use either of these options you have to select what version the model you're loading is. The captions tool is a quick and easy way to create text files containing captions to train each image on.

**Reply by i-huzaifa-arshad** [reply]
Can you please explain how to work with captions? Or if there is any tutorial, that would be best. Tahnks.

**Reply by Excalibro1** [reply]
This is most likely the best you'll get in that regard atm. 

https://www.reddit.com/r/StableDiffusion/comments/zcr644/make_better_dreambooth_style_models_by_using/?utm_source=share&utm_medium=android_app&utm_name=androidcss&utm_term=1&utm_content=share_button
